deploy/operator.yaml

```diff
@@ -30,3 +30,5 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "mobile-developer-console-operator"
+            - name: OPENSHIFT_HOST
+              value: "${OPENSHIFT_HOST}"  # to be filled by whoever is deploying this file
```

Comment:
Do you think it would be useful to change this to be an openshift "Template", rather than a "Deployment"? That way, this could be set as a parameter, and it would be more difficult for someone to accidentally miss it. I'm not sure if it's worth the effort, just a thought.

Reply:
IMO, let's leave as is for now as we're mainly going to install this thing over RHMI and kind of hard to miss there once it is added to installer.

Reply:
Hi @aliok could we inject the OPENSHIFT_HOST value via the CR rather than the operator.yml/template as it seems to be specific to the mdc config rather than the operator pod right? Also it would be nice for the user not to have to configure this at all, is there any way to get it internally?

Reply:
@laurafitzgerald it seems like a strange thing to define in the CR, since we'd never want someone creating a CR to enter anything other than "this cluster", right? There's no way that we're aware of to get it internally (you can get the internal address, but not the external domain).

Reply:
👍 and thanks for the extra detail provided.

Reply:
:+1: 

>  it seems like a strange thing to define in the CR, since we'd never want someone creating a CR to enter anything other than "this cluster", right?

... until Kubernetes people go nuts and come up with a concept like *cross cluster resources* :grin: